raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -48,6 +48,7 @@
 import org.apache.kafka.common.requests.DescribeQuorumResponse;
 import org.apache.kafka.common.requests.EndQuorumEpochRequest;
 import org.apache.kafka.common.requests.EndQuorumEpochResponse;
+import org.apache.kafka.common.requests.FetchRequest;
```

Comment:
Not related to this line. It would be better to always construct the FetchRequest with the new schema [here](https://github.com/apache/kafka/blob/a304d91d49b2ca56b51d3a9d6589ef69a6065bbb/raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java#L1791) and to downgrade in the builder later on.